packages/web/src/pages/upload-page/forms/EditTrackForm.tsx

```diff
@@ -180,18 +181,16 @@ const MultiTrackFooter = () => {
   const nextDisabled = index === trackMetadatas.length - 1
   return (
     <div className={cn(styles.multiTrackFooter, layoutStyles.row)}>
-      <HarmonyButton
+      <HarmonyPlainButton
         className={cn({ [styles.disabled]: prevDisabled })}
-        variant={HarmonyButtonType.PLAIN}
         text={messages.prev}
         iconLeft={IconCaretLeft}
         onClick={goPrev}
         disabled={prevDisabled}
         type='button'
       />
-      <HarmonyButton
+      <HarmonyPlainButton
         className={cn({ [styles.disabled]: nextDisabled })}
```

Comment:
kinda the same here, like we should just have a disabled prop. if we just want to do follow-ons to the butons when we do harmony, mind just adding a ticket so we have it covered?